.gitignore

```diff
@@ -3,5 +3,7 @@ target/
 user/
 figures/
 vizlab/
-makefile
+[Mm]akefile
 .Rproj.user
+.Rhistory
+internal-analytics.Rproj
```

Comment:
Right. I think I'm saying the same thing. Our group has traditionally ignored it, but this:

![image](https://user-images.githubusercontent.com/1105215/27042991-46bc9f58-4f5e-11e7-8637-f7f9c8a54e40.png)

Might be a good reason for us to reverse that policy